languages/jetpack-af.po

```diff
@@ -34,7 +34,7 @@ msgstr ""
 
 #. translators: %s is a URL
 #: class.jetpack.php:1413
-msgid "In <a href=\"%s\" target=\"_blank\">Development Mode</a>, via site URL lacking a dot (e.g. http://localhost)."
```

Comment:
For the PO files, I'd vote removing them from the PR and let the normal translation process handle the string updates.